c7n/filters/iamaccess.py

```diff
@@ -50,6 +50,8 @@ class PolicyChecker:
     checker_config:
       - check_actions: only check one of the specified actions
       - everyone_only: only check for wildcard permission grants
+      - condition_scope: cross-account scope of conditions that are
```

Comment:
the name on the field and the doc make it a little hard to infer, what's it doing. I don't have better name in mind, but the doc here could be improved.